### PR TITLE
Fix deploy group-level JVM resource.

### PR DESCRIPTION
**Pull request by rahulsayini**
Before you contribute, please review these guidelines to help ensure a smooth process for everyone.

Thanks.

* Read [how to properly contribute to open source projects on Github][1].
* Fork the project.
* Use a feature branch.
* Write [good commit messages][2].
* Use the same coding conventions as the rest of the project.
* Commit locally and push to your fork until you are happy with your contribution.
* Make sure to add tests and verify all the tests are passing when merging upstream.
* Add an entry to the [Changelog][3] accordingly.
* Please add your name to the [CONTRIBUTORS.md][7] file. Adding your name to the [CONTRIBUTORS.md][7] file signifies agreement to all rights and reservations provided by the [License][4].
* [Squash related commits together][5].
* Open a [pull request][6].
* The pull request will be reviewed by the community and merged by the project committers.

[1]: http://gun.io/blog/how-to-github-fork-branch-and-pull-request
[2]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[3]: ./CHANGELOG.md
[4]: ./LICENSE
[5]: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
[6]: https://help.github.com/articles/using-pull-requests
[7]: ./CONTRIBUTORS.md

### CODE REVIEW

File: jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java [hunk 1]
```diff
@@ -439,17 +443,17 @@ public Group generateAndDeployGroupJvmFile(final String groupName, final String
             futures.put(jvm.getJvmName(), future);
         });
 
-        checkResponsesForErrorStatus(futures);
+        checkJvmGenerateAndDeployFutureErrorStatus(futures);
         return group;
     }
 
-    private void checkResponsesForErrorStatus(final Map<String, Future<Response>> futureMap) {
+    private void checkJvmGenerateAndDeployFutureErrorStatus(final Map<String, Future<Jvm>> futureMap) {
```
Comment (akinny) [inline jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java]: Add comments/documentation on what this method does. Let's keep this as a practice going forward

**Review by akinny — APPROVED** [review]

File: jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java [hunk 2]
```diff
@@ -463,12 +467,6 @@ private void checkResponsesForErrorStatus(final Map<String, Future<Response>> fu
                     errorMap.put(key, Collections.singletonList(e.getMessage()));
                 }
             }
-
-            if (response != null && response.getStatus() > 399) {
```
Comment (jeddcuison) [inline jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java:467]: Just curious, why don't we check the response status anymore ?

File: jwala-tomcat/build.gradle [hunk 3]
```diff
@@ -1,176 +1,177 @@
-description "This module generates and deploys Jwala binaries. This layes creates the zip file which can be distributed and published to a repository for releases"
```
Comment (jeddcuison) [inline jwala-tomcat/build.gradle]: Why does git flag the whole content ? Can you double check hidden characters ? e.g. tabs = 4 spaces

**Review by jeddcuison — COMMENTED** [review]
Looks good but I have 2 questions can you please address them ? Thanks!

File: jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java (hunk 2 above)
Comment (rahulsayini) [reply jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java:467]: This is at service layer. We do the response check at rest layer.

**Pull request merged**